<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="92067" started-at="2017-01-18T06:28:02Z" finished-at="2017-01-18T06:29:34Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="92067" started-at="2017-01-18T06:28:02Z" finished-at="2017-01-18T06:29:34Z">
      <class name="testCases.SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality">
        <test-method status="PASS" signature="setSnapShotFolder()[pri:0, instance:testCases.SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality@43bd930a]" name="setSnapShotFolder" is-config="true" duration-ms="32" started-at="2017-01-18T11:58:02Z" finished-at="2017-01-18T11:58:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setSnapShotFolder -->
        <test-method status="PASS" signature="BeforeMethod()[pri:0, instance:testCases.SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality@43bd930a]" name="BeforeMethod" is-config="true" duration-ms="17966" started-at="2017-01-18T11:58:02Z" finished-at="2017-01-18T11:58:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeMethod -->
        <test-method status="FAIL" signature="main()[pri:0, instance:testCases.SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality@43bd930a]" name="main" duration-ms="72527" started-at="2017-01-18T11:58:20Z" finished-at="2017-01-18T11:59:33Z">
          <exception class="org.openqa.selenium.StaleElementReferenceException">
            <message>
              <![CDATA[Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 20.09 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'surender3145693', ip: '10.175.172.32', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_05'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=46.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 67e4af97-3414-44de-af5e-052ffb200aaa]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 20.09 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'surender3145693', ip: '10.175.172.32', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_05'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=46.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 67e4af97-3414-44de-af5e-052ffb200aaa
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:85)
	at testCases.SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality.main(SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'surender3145693', ip: '10.175.172.32', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_05'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.cache.getElementAt(resource://fxdriver/modules/web-element-cache.js:9454)
	at <anonymous class>.Utils.getElementAt(file:///C:/Users/SUREND~1/AppData/Local/Temp/anonymous3148317617097025982webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:9039)
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/SUREND~1/AppData/Local/Temp/anonymous3148317617097025982webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10090)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/SUREND~1/AppData/Local/Temp/anonymous3148317617097025982webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12644)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/SUREND~1/AppData/Local/Temp/anonymous3148317617097025982webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/SUREND~1/AppData/Local/Temp/anonymous3148317617097025982webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:625)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.StaleElementReferenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testCases.SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality@43bd930a]" name="afterMethod" is-config="true" duration-ms="1053" started-at="2017-01-18T11:59:33Z" finished-at="2017-01-18T11:59:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testCases.SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality@43bd930a]" name="tearDown" is-config="true" duration-ms="10015" started-at="2017-01-18T11:59:34Z" finished-at="2017-01-18T11:59:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.SS_CartPage_Private_Verify_CartPage_ApplyCoupon_Functionality -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
